governance-exercises/Le Bateleur - Poll Exercise 02/Course of Action.md

```diff
@@ -0,0 +1,61 @@
+
+There is also the matter of a whistleblower and how the breach-of-anonymity evidence was presented. In the first of the two instances, the identities of the Aligned Delegates were already known, since the delegation system predated Endgame's anonymity requirement. In the second instance, there appears to have been no whistleblower; and the breach-of-anonymity evidence was investigated privately by the Governance Facilitator.
+
+The immediate scenario presents differently. Diane posted the materials publicly on the Forum. Clive is challenging the verity of the evidence. Critically, it is unclear whether the breach-of-anonymity evidence submitted by Diane rises to the threshold specified in [Atlas 2.6.6](https://mips.makerdao.com/mips/details/MIP101#2-6-6-aligned-delegate-privacy): "If there’s **clear evidence or significant suspicion** [emphasis added] that the identity of an AD has been revealed, a FacilitatorDAO must immediately derecognize the AD." 
+
+The language in Atlas 2.6.6 is ambiguous. While "clear evidence" presents no issue, the phrase "significant suspicion" presents a gray area. Is "reasonable suspicion" equivalent to the "significant suspicion" threshold that we find in Atlas 2.6.6? There's no clear answer based on the text alone. But I am also aware of my [mandate](https://mips.makerdao.com/mips/details/MIP999#2-4-2-alignment-conservers-eligibility-requirements) to minimize risk to Maker and guard against misalignment threats. This is a close call, especially given the ambiguity in Atlas 2.6.6. I am compelled in this instance to err on the side of caution and determine that the materials provided by Diane *do* meet the specified threshold of "significant suspicion."
+
+But a determination from the Governance Facilitator is not to be the final word. Per [Atlas 2.6.6.1.1](https://mips.makerdao.com/mips/details/MIP101#2-6-6-1-1), a review of the decision by all FacilitatorDAOs is required. Because FacilitatorDAOs don't yet exist, running a governance poll can serve as a crucial "check" on the Governance Facilitator's power. There is also (outdated) [historical support](https://forum.makerdao.com/t/mip102c2-sp2-mip-amendment-subproposals/20016/19#h-1011-constitutional-boundaries-of-the-arbitration-scope-framework-67) for using a governance poll to adjudicate derecognition matters. (The former Maker Constitution, 10.11.3.4, states, "During the Pregame, all actions of derecognition must be decided through a MKR governance poll...")
```

Comment:
Not a line of thought I've seen in the exercises thus far. I had discounted that review clause given FacilitatorDAOs don't exist yet. 

Honestly, I was probably working on assumptions from previous versions of the constitution/atlas.